fs_xgspon_mod.py

```diff
@@ -604,7 +604,7 @@ def finish_request(self, request, client_address):
 
 def persist(args):
     try:
-        with CigTelnet(args.onu_ip, args.serial) as tn:
+        with CigTelnet(args.onu_ip, args.fs_onu_serial) as tn:
```

Comment:
This one should be `isp_ont_serial` (sorry to quibble, just not at a desktop right now)